client/migrations/0002_auto_20200326_1409.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.0.4 on 2020-03-26 21:09
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('client', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='volunteer',
+            name='first_name',
+            field=models.CharField(default='abcdef', max_length=200),
+        ),
+        migrations.AddField(
+            model_name='volunteer',
+            name='last_name',
+            field=models.CharField(default='abcdef', max_length=200),
```

Comment:
maybe make that blank=True field for now? we shouldn't use the abcdef default here

Reply:
According to app logic each person must have a first name and a last name, so blank=True seems like it violates our goals. Django does not seem to allow non-blank fields without default values, hence the abcdef...

Reply:
I see, so ignore the blank=True suggestion
but what about setting default to an empty string?

Reply:
Could work better, I suppose. I'll make the change.